x-pack/filebeat/module/awsfargate/log/test/nginx.log-expected.json

```diff
@@ -0,0 +1,20 @@
+[
+    {
+        "event.dataset": "awsfargate.log",
+        "event.module": "awsfargate",
+        "event.original": "10.0.1.195 - - [24/Sep/2020:16:45:00 +0000] \"GET /config/getuser?index=0 HTTP/1.1\" 404 153 \"-\" \"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:76.0) Gecko/20100101 Firefox/76.0\" \"45.148.10.28\"",
+        "fileset.name": "log",
+        "input.type": "log",
+        "log.offset": 0,
+        "service.type": "awsfargate"
+    },
+    {
+        "event.dataset": "awsfargate.log",
+        "event.module": "awsfargate",
+        "event.original": "10.0.1.195 - - [24/Sep/2020:16:45:19 +0000] \"GET / HTTP/1.1\" 200 612 \"-\" \"ELB-HealthChecker/2.0\" \"-\"",
+        "fileset.name": "log",
+        "input.type": "log",
+        "log.offset": 189,
+        "service.type": "awsfargate"
+    }
```

Comment:
I'm wondering, is cloudwatch providing any extra meta about the source of the logs? I would expect some info about the containers generating these logs

Reply:
Hmm the only metadata I can find is the container ID, which is in the name of the log stream. For example: `ecs/metricbeat-awsfargate/397eb2787a7d4f7783d03c49cafd244c`.